### PR TITLE
#18292 Merging amp branch

### PR DESCRIPTION
**Pull request by rafaelzufil**

### CODE REVIEW

File: plugins/cli.dat [hunk 1]
```diff
@@ -6,3 +6,11 @@ health-check
...
 wp-user-avatar
 pushcrew
+amp
+autoptimize
+cache-enabler
+hummingbird-performance
+regenerate-thumbnails
+shins-pageload-magic
+webp-express
+wp-smushit
```
Comment (gmenoiaa) [inline plugins/cli.dat]: add a new line

File: plugins/linked.dat [hunk 2]
```diff
@@ -0,0 +1 @@
+amp-google-tag-manager
```
Comment (gmenoiaa) [inline plugins/linked.dat]: add a new line

File: plugins/cli.dat [hunk 3]
```diff
@@ -6,3 +6,11 @@ health-check
 wp-custom-post-template
 wp-user-avatar
 pushcrew
+amp
+autoptimize
+cache-enabler
+hummingbird-performance
```
Comment (gmenoiaa) [inline plugins/cli.dat]: John mentioned they want to reduce the number of plugins, so can we just include the one's which will be enabled?

Comment (balazsbohonyi) [reply plugins/cli.dat]: I agree - most of the disabled plugins where only used prior to amp changes. I would maybe include Hummingbird from the disabled plugins as it helps with DB optimization and checking server GZIp / browser caching config

Comment (gmenoiaa) [reply plugins/cli.dat]: ok, so we can remove:

`cache-enabled`
`autoptimize`
`shins-pageload-magic`

correct?

**Pull request merged**